### DISCUSSION TITLE
How to store fonts used in plotly charts

### DISCUSSION THREAD

**Discussion by MaxGhenis** [executablebooks/meta#187] [Q&A]
I'm using the Roboto font in plotly charts in my jupyter books, e.g. [this one](http://child-allowance.ubicenter.org/). It looks right on my computer, as I have Roboto installed, but for others it shows up as Times New Roman. Is there a way to store the font files so it shows up correctly for all viewers?

**Comment by choldgraf** [comment]
Hmm that’s a good question, have you played around with different renderers in plotly? I think some of them embed all the data needed to recreate the plot while others assume some libraries or css is preloaded, maybe one of them would solve your fonts issue as well?
